spec/cookbook_spec.rb

```diff
@@ -170,13 +170,26 @@
       :result => [
       ],
     },
+    {
+      :name => 'match deepest subfolder first',
+      :files => [
+        {
+          :status => :modified,
+          :path => 'cookbooks/core/cb_one/metadata.rb',
+        },
+      ],
```

Comment:
this test doesn't show that it matches it first, only that it matches it, right?